Akka.Serialization.MessagePack/Formatters/SerializableFormatter.cs

```diff
@@ -0,0 +1,71 @@
+#if SERIALIZATION
+using System.Reflection;
+using System.Runtime.Serialization;
+using System.Text;
+using Akka.Util.Internal;
+using MessagePack;
+using MessagePack.Formatters;
+
+namespace Akka.Serialization.MessagePack.Resolvers
+{
+    public class SerializableFormatter<T> : IMessagePackFormatter<T>
```

Comment:
What is this for?

Reply:
IDK, there was a `Serializable` formatter in the old repo... I'm guessing it's mostly normally used for exceptions?

IOW This is brought over from the old version, but kept along b/c I feel like it -may- be needed. But I'm honestly not sure.